src/EntryPoint/RestEntryPoints.ts

```diff
@@ -52,29 +52,27 @@ export class RestEntryPoints implements EntryPoints {
     });
   }
 
-  public async entrypoints(
-    projectId: string,
-    limit: number = 10,
-    connectivity?: string[],
-    status?: string[]
-  ): Promise<EntryPoint[]> {
-    let remaining = limit;
+  public async entrypoints(filter: EntryPointFilter): Promise<EntryPoint[]> {
+    let remaining = filter.limit ?? 10;
```

Comment:
Use destructuring assignment syntax wherever possible for better code clarity
```suggestion
  public async entrypoints({ limit = 10, projectId, ...filters }: EntryPointFilter): Promise<EntryPoint[]> {
    let remaining = limit;
```